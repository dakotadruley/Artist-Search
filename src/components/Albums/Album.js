import React from 'react';
import PropTypes from 'prop-types';

const Album = ({ title, date }) => (
  <section>
    <h2>{title}</h2>
    <p>{date}</p>
  </section>
);

Album.propTypes = {
  id: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
  date: PropTypes.string.isRequired
};

export default Album;

// http://coverartarchive.org/release/<RELEASE_ID>/front
// don't make a fetch call for cover art, it's conditional and you need the albulm id 
