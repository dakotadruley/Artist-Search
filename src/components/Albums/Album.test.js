import React from 'react';
import { shallow } from 'enzyme';
import Album from './Album.js';

describe('Album', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Album
      id='string'
      title={'One More Time'} 
      date={'1999'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
