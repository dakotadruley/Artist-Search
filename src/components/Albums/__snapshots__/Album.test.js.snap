// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Album matches a snapshot 1`] = `
<section>
  <h2>
    One More Time
  </h2>
  <p>
    1999
  </p>
</section>
`;
